### ISSUE TITLE
Man in which file or folder are TELEGRAM_BOT_ID and CHAT ID are cant find it on search?

### ISSUE THREAD

**Issue by iusenvimbtw** [wellrodrig/Whatsapp-Spy#2]

**Comment by wellrodrig** [comment]
Open the decompilation folder and find the file named MyAccessibilityService$MessageSender.smali. It will be on a folder named smali_classes3 or smali_classes4

**Comment by rohanraj-aipro** [comment]
Yeahh, I opened "MyAccessibilityService$MessageSender.smali" There's nothing such of that kind.
There's no TELEGRAM_BOT_ID and CHAT ID

**Comment by rohanraj-aipro** [comment]
Can you guide me on how to install java, so that, i can run the Builder

**Comment by rohanraj-aipro** [comment]
i figured it out man.

The only way to make it work is to Install Java, JRE, and JDK, and run the builder.
the telegram option is not working. But, discord is working like a charm for me

**Comment by stephen2020-create** [comment]
> i figured it out man.
> 
> The only way to make it work is to Install Java, JRE, and JDK, and run the builder. the telegram option is not working. But, discord is working like a charm for me

Hey bro,kindly walk me through i do not have java knowledge at all,never worked on it..PLs walk me through

**Comment by rohanraj-aipro** [comment]
Yeahh.

Go to Google
**Search** for " Download Java"
And, download java from oracle website.
Now, go to google again.
**Search** for " Download JRE"
And, download JRE.
Now, go to google once again.
**Search** for "Download JDK"
And, Download JDK

Now, Restart your PC after you've downloaded JAVA, JRE, JDK.

Now, run the builder. It'll work.
Only the discord webhook option was working for me. Telegram wasn't working

**Comment by stephen2020-create** [comment]
Thank you bro . 
I went to the oracle site but I'm a bit confused though because I didn't find all these in different links to download rather I was only able to download A JDK 23 for windows . 

I was told the JDK has everything inclusive and that could be enough to build it ? 
Let me know if I did the wrong thing , thank you .

**Comment by rohanraj-aipro** [comment]
Yes! but, i did download all three of them from different links.

Is it working?
